src/adapters/gimel_adapter.js

```diff
@@ -0,0 +1,25 @@
+import TrackingAdapter from "./tracking_adapter"
+import utils from "../utils"
+
+// # Adapter for using the gimel backend. See https://github.com/Alephbet/gimel
+// # The main difference is the user_uuid generation (for backwards compatibility)
+// # params:
+// # - url: Gimel track URL to post events to
+// # - namepsace: namespace for Gimel (allows setting different environments etc)
+// # - storage (optional) - storage adapter for the queue
+class GimelAdapter extends TrackingAdapter {
+  static queue_name = "_lamed_queue"
```

Comment:
this should be `_gimel_queue`? was this bug on the original code?

Reply:
Yes, it was like this already in the original code. I will change it.

Reply:
Updated it.